.github/workflows/python-app.yml

```diff
@@ -53,27 +53,34 @@ jobs:
     - name: djangosaml2 SP metadata to Proxy
       run: |
         wget http://localhost:8000/saml2/metadata -O example/metadata/sp/djangosaml2_sp.xml
+    - name: Create docker-example folder
+      run: |
+        mkdir -p docker-example
+        cp -r example/. docker-example
     - name: run satosa-saml2spid
       run: |
-        cd example
+        cd docker-example
         mkdir -p metadata/idp
         mkdir -p metadata/sp
         export SATOSA_APP=`python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])'`
-        uwsgi --wsgi-file $SATOSA_APP/satosa/wsgi.py  --https 0.0.0.0:10000,./pki/cert.pem,./pki/privkey.pem --callable app -b 32768 &
-        sleep 5
+    - name: run docker compose
+      run: |       
+        cd Docker-compose
+        docker compose -f docker-compose.yml up -d
+        sleep 5    
```

Comment:
is this sleep still required?

Reply:
a "sleep" never hurts :) 
would you prefer it to be removed?